examples/lang/erc20/src/lib.rs

```diff
@@ -17,17 +17,16 @@
 #![cfg_attr(not(any(test, feature = "test-env")), no_std)]
 
 use ink_core::{
-    env::{
-        self,
-        AccountId,
-        Balance,
-    },
     memory::format,
     storage,
 };
+use ink_model::EnvHandler;
 use ink_lang::contract;
+use ink_types_node_runtime::NodeRuntimeTypes;
 
 contract! {
+    #![env = NodeRuntimeTypes]
```

Comment:
Just to clarify: We could also write `#![env = ink_types_node_runtime::NodeRuntimeTypes]`, right?

Reply:
Correct, that part is parsed as a `syn::Type`. e.g. https://github.com/paritytech/ink/pull/108/files#diff-18215f9b77a1fc56d4103de0bf553be0R11